asif.md

```diff
@@ -0,0 +1,5 @@
+name:Mohd Asif Khan 
```

Comment:
Create a folder inside `_data` called `asif` and move this file inside that folder, resulting `_data/asif/asif.md`.